readthedocs/settings/base.py

```diff
@@ -297,6 +297,8 @@ def INSTALLED_APPS(self):  # noqa
             "allauth.socialaccount.providers.gitlab",
             "allauth.socialaccount.providers.bitbucket_oauth2",
             "allauth.mfa",
+            # Others
+            "impersonate",
             "cacheops",
```

Comment:
Unrelated: we can remove `cacheops` as we never used it in the end since we made our db queries to work faster instead.